### PR TITLE
Homework

### PR DESCRIPTION
**Pull request by Laura829**
First HW

### CODE REVIEW

**Comment by Laura829** [comment]
A title leírásánál head helyett body-t írtam :D

File: homework.html [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+</p>
+<p>
+        <em>Válassz ki egy szövegszintű, egy csoportosító és egy szekciót képező elemet a HTML Living Standardból és írj egy kb. 5 mondatos összefoglalót az egyes elemekről.</em>
+</p>
+<p>
+        <strong>Szövegszintű/Text level:</strong><br>
+        &ltstrong&gt&lt/strong&gt<br>
+        Amit látunk: félkövérré teszi a körbeölelt szót.<br>
+        Egy szövegben a fontosabb szavakat emelhetjük ki általa.<br>
+        Ahány strong van egymás mellett annál erősebb a nyomatékosítás.<br>
```
Comment (szcsl) [inline homework.html]: My bad, this was meant to be a joke during the lecture, sorry. :D Multiple levels of strong elements will not be louder. My fault completely, will not joke with technology next time. :)

File: asideexample.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        peace and love at least four times, so I’m promoting peace and love on the record.<br>
+        But I’m having a bit of fun with rock and roll and we have a great ballad.<br>
+        I also write with other people on this one. If you backtrack to the other records,<br>
+        there’s usually ten tracks that I’ve written with someone else, this one has five.<br>
+        There’s also a cover of an old song, “Money (That’s What I Want).”<br>
+        I wanted to record “Money” and I wanted to do it my way.<br>
+        That’s the joy of making your own record: We wanted to do it, so we did.</q><br>
+    </p>
+    <aside style="font-size: 18pt;">
+        <strong><em><q>If I felt like that, I would have made one record. On this record, we mention<br>
```
Comment (szcsl) [inline asideexample.html]: Using strong and em this way is not semantic. The distinction is not necessarily straightforward, but here's a good example: https://stackoverflow.com/questions/1936864/strong-vs-em

File: homework.html (hunk 1 above)
Comment (Laura829) [reply homework.html]: I'm crying 🤣 Amúgy volt egy sejtésem... 🤣

**Pull request merged**